sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -376,3 +376,11 @@
   min-height: 150px;
 }
 
+.img-student-welcome-background {
+  background-image: url("../images/image_student_welcome.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 100%;
+  height: 100%;
+  min-height: 250px;
```

Comment:
i see min height `250px`  used in other classes too, 
![image](https://user-images.githubusercontent.com/77353155/154749666-2f526420-d3b5-42a1-9a79-702d27d35c66.png)
. maybe we can create a scss constant for it

Reply:
Is it really common across all images? Is it something that does not change depending on where the image is placed?